src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilterTest.java

```diff
@@ -179,4 +182,17 @@ private static SuppressionXpathFilter createSuppressionXpathFilter(String fileNa
         return suppressionXpathFilter;
     }
 
+    @Test
+    public void testAcceptThree() throws Exception {
```

Comment:
@strkkk Sir, can you please explain how this check work. On whatever input I give, it always asserts true which should not be valid.

Reply:
I got it. Added tests for both String Literal and Char literal.